pyglossary/plugins/cc_cedict/conv.py

```diff
@@ -29,10 +29,10 @@ def parse_line(line):
 	return trad, simp, pinyin, eng
 
 
-def make_entry(trad, simp, pinyin, eng):
+def make_entry(trad, simp, pinyin, eng, trad_name):
 	eng_names = list(map(summarize, eng))
-	names = [simp, trad, pinyin] + eng_names
-	article = render_article(trad, simp, pinyin, eng)
+	names = [trad if trad_name else simp, simp if trad_name else trad, pinyin] + eng_names
```

Comment:
Confusing. Please put each of the 3 items in a new line.

```
names = [
	trad if trad_name else simp,
	simp if trad_name else trad,
	pinyin,
] + eng_names
```

Reply:
Done.